locales/en-US/experiments.ftl

```diff
@@ -143,7 +142,6 @@ snoozetabsContributors5Title = Firefox QA
 snoozetabsContributors6Title = Softvision QA
 snoozetabsContributors7Title = Softvision QA
 tabcenterDescription = What would it be like to move tabs from the top of the browser to the side? We wanted to find out!
-tabcenterWarning = Firefox is changing fast. Tab Center may have bugs in Firefox 55 and will be disabled completely in Firefox 56+.
```

Comment:
Looks like this one was added manually to the automatically generated FTL - it was never added to the YAML.